website/next.config.js

```diff
@@ -1,4 +1,12 @@
 import nextra from 'nextra'
+import path from 'node:path'
+import { visit } from 'unist-util-visit'
+
+import { translate } from '@edgeandnode/gds'
+
+// Compile `i18n.ts` to `i18n.js` since we can't import `ts` files in `next.config.js`
+import { translations } from './dist/i18n.js'
```

Comment:
😄 Next config will support TypeScript in v15: https://github.com/vercel/next.js/discussions/35969#discussioncomment-10005034

Reply:
Amazing!! didn't know that 😍